letters/modernized/16-letters.adoc

```diff
@@ -450,16 +472,22 @@ manifested in and through his beloved Son Jesus Christ, to whom be ascribed glor
 salvation and honour.
 Amen.
 
+[.signed-section-closing]
 Your friend and sister, I hope, in the saints`' travail and tribulation,
```

Comment:
On my PDF viewer, this dropped down onto a second line. I wonder if our viewers are a little bit off.  Jared says its possible.  Why don't we bump this back up as part of the paragraph above. 👍🏻